frontend/src/component/common/Table/cells/StringArrayCell.tsx

```diff
@@ -0,0 +1,58 @@
+import { VFC } from 'react';
+import { styled, Typography } from '@mui/material';
+import { TextCell } from 'component/common/Table/cells/TextCell/TextCell';
+import { Highlighter } from 'component/common/Highlighter/Highlighter';
+import { useSearchHighlightContext } from 'component/common/Table/SearchHighlightContext/SearchHighlightContext';
+import { TooltipLink } from 'component/common/TooltipLink/TooltipLink';
+
+const StyledTag = styled(Typography)(({ theme }) => ({
+    fontSize: theme.fontSizes.smallerBody,
+}));
+
+interface IArrayFieldCellProps<T> {
+    row: T;
+    field: keyof T;
+    singularLabel: string;
+}
+
+export const StringArrayCell: VFC<IArrayFieldCellProps<any>> = ({
+    row,
+    field,
+    singularLabel,
+}) => {
+    const { searchQuery } = useSearchHighlightContext();
+    const fieldValue = row[field];
+
+    if (!Array.isArray(fieldValue) || fieldValue.length === 0)
+        return <TextCell />;
+
+    const pluralLabel = `${singularLabel}s`;
```

Comment:
This is an edge case, I guess, but not all nouns pluralize by adding an "s" at the end. "Shrimp", "sheep", and "fish" are some examples. Additionally, some words change vowels (e.g. "hobby" becomes "hobbies"). It might be okay in our current system, but it might also be incorrect sometimes. I would suggest that you either have to supply a pluralized label too or that you *can* provide that label, and if you don't the system falls back to just adding the s.

Reply:
Yes, I thought about that, but thought to fix it when we run into that case, but I can make it right now.